x-pack/plugins/endpoint/public/applications/endpoint/store/policy_list/fake_data.ts

```diff
@@ -29,17 +37,22 @@ const getRandomNumber = () => {
   return randomNumbers[randomIndex];
 };
 
+const getRandomId = () => {
+  return Math.floor(Math.random() * 100000).toString();
+};
+
 export const getFakeDatasourceApiResponse = async (page: number, pageSize: number) => {
   await new Promise(resolve => setTimeout(resolve, 500));
 
   // Emulates the API response - see PR:
   // https://github.com/elastic/kibana/pull/56567/files#diff-431549a8739efe0c56763f164c32caeeR25
-  return {
+  const policyData = {
     items: Array.from({ length: pageSize }, (x, i) => ({
```

Comment:
Maybe extract the iterator function here to a top-level function `generateSinglePolicyData(name)` and then you can use it in `getFakeDatasourceDetailsApiResponse()` without the hackery 😃  ?

Reply:
I know I can do that, I was mostly using the hackery to so that when I click on a Policy name, the same name will show up on the Policy details (from the same dataset).  I'll adjust the code though so that this isn't so hacky.

Reply:
Ahh. Ok. In that case, then maybe tie the `id` to the name in the generator? (instead of random strings, maybe use the `pageNumber` + `pageSize` to get consistent results?
If too much trouble, then just leave it as it - it is "fake" data 😆

Reply:
made to be less hacky now :)